Assets/Editor/ScheduledEventTest.cs

```diff
@@ -0,0 +1,182 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System;
+using NUnit.Framework;
+using Scheduler;
+using UnityEditor;
+using UnityEngine;
+
+public class ScheduledEventTest
+{
+    private Action<ScheduledEvent> callback;
+    private bool didRun = false;
+
+    [SetUp]
+    public void Init()
+    {
+        callback = (e) =>
+            {
+                Debug.ULogChannel("ScheduledEventTest", "Event {0} fired", e.Name);
+                didRun = true;
+            };
+    }
+
+    [Test]
+    public void EventCreationTest()
+    {
+        ScheduledEvent evt = new ScheduledEvent(
+            "test",
+            (e) => Debug.ULogChannel("ScheduledEventTest", "Event {0} fired", e.Name),
+            3.0f,
+            true,
+            1);
+
+        ScheduledEvent evt2 = new ScheduledEvent(evt);
+
+        Assert.That(evt.Name, Is.EqualTo("test"));
```

Comment:
IMO I think `Assert.That(x, Is.EqualTo(y));` reads just like English, so its easier to understand at a glance